models/articles.models.js

```diff
@@ -31,3 +31,23 @@ exports.selectArticles = () => {
     return result.rows;
   });
 };
+
+exports.selectCommentByArticleId = (article_id) => {
+  return db
+    .query(
```

Comment:
You need to check if the article exists in order to determine whether the empty array being returned is due to there being no comments, or due to the article not existing.  Think back to Stephens Tuesday lecture!